define/types.go

```diff
@@ -75,8 +74,7 @@ var (
 		"CAP_SETUID",
 		"CAP_SYS_CHROOT",
 	}
-	// DefaultNetworkSysctl is the list of Kernel parameters which we
-	// grant by default to containers which are running under UID 0.
+	// DefaultNetworkSysctl is deprecated use values from containers.conf
```

Comment:
This should be written as `// Deprecated: ...`, see https://stackoverflow.com/questions/7849663/how-do-you-mark-code-as-deprecated-in-go

There is a linter which will then catch this and warn users about it same in the other places.